src/components/Project/RedeemTokenViaExchange.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { CSSProperties, useContext } from 'react'
+import { Tooltip } from 'antd'
+import AMMPrices from 'components/AMMPrices'
+import { ThemeContext } from 'contexts/themeContext'
+
+export const RedeemTokenViaExchange = ({
+  tokenSymbol,
+  tokenAddress,
+  style,
+}: {
+  tokenSymbol: string
+  tokenAddress: string
+  style?: CSSProperties
+}) => {
+  const {
+    theme: { colors },
+  } = useContext(ThemeContext)
+
+  return (
+    <span style={style}>
+      <Tooltip
```

Comment:
**Suggestion:** I think we should include an `or` here like in the `PayInputSubText` 

<img width="180" alt="Screen Shot 2022-09-24 at 1 55 16 pm" src="https://user-images.githubusercontent.com/96150256/192078994-3e5f987e-9305-40f4-a607-318f0bdb687a.png">

Reply:
(Then lowercase case the 'R' in 'Redeem' below)

Reply:
affirmative ser!